pysyncrosim/project.py

```
@@ -297,25 +297,40 @@ def datasheets(self, name=None, summary=True, optional=False, empty=False,
                                                     return_hidden, self.pid)
         return self.__datasheets
     
-    def delete(self, scenario=None, force=False):
+    def delete(self, scenario=None, data=None, datasheet=None, ids=None,
```

Ruff (0.14.11)

Unused method argument: `scenario`

(ARG002)